Aztec/Classes/TextKit/TextView.swift

```diff
@@ -242,12 +242,22 @@ open class TextView: UITextView {
             .font: defaultFont,
             .paragraphStyle: defaultParagraphStyle,
         ]
-        if let color = textColor {
+        if let color = defaultTextColor {
             attributes[.foregroundColor] = color
         }
         return attributes
     }
-    
+
+    open var defaultTextColor: UIColor?
```

Comment:
Should we have a "smart" `defaultColor` pre-set?
This way we could avoid the need to always set a defaultTextColor on every place we use Aztec, and just set it if we want a different default.

Reply:
Good suggestion, I'm going to change it.

Reply:
Addressed by: https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1237/commits/ba8524aba1332550efb05cad583a85ed3511beb5